src/nanoemoji/write_font.py

```diff
@@ -784,6 +784,9 @@ def _generate_color_font(config: FontConfig, inputs: Iterable[InputGlyph]):
     ttfont = _make_ttfont(config, ufo, color_glyphs)
 
     if ttfont is not None:
+        # apply_ttfont may wish to do things like reorder tables that require full load
+        ttfont = util.load_fully(ttfont)
```

Comment:
wouldn't it be better to only do that if we are going to actually reorder glyphs? it's a bit expensive operation (especially if we do keep the reload step)

Reply:
Currently we don't know if the next step reorders. I suppose we could push down the reorder and have apply_ttfont return a font that replaces the one passed in.